Sources/ArcGISToolkit/Extensions/String.swift

```diff
@@ -0,0 +1,43 @@
+// Copyright 2023 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+extension String {
```

Comment:
Instead of putting these on `String`, consider that most—if not all—uses of these either directly or indirectly create an instance of `Text`. So if they were moved into an extension of `Text`, then they would be even simpler to use:

```swift
extension Text {
    static var cancel: Self {
        ...
    }
}
```

It might take a little refactoring in some places (when the string is passed around before creating the text, change the type of the parameter/property for the string to `Text`), but it is worth investigating at least.

Reply:
#421